src/components/AccountPage/TokensList.js

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
Maybe rename this kind of files to `SubAccountList` then

Reply:
true. we'll do that when we rework the UI/UX with design team. to also generalize that UI on their side.